$NetBSD: patch-fiptool_uuid_parse,v 1.1 2019/05/14 04:08:51 thorpej Exp $

Patch from upstream to fix UUID parsing bug in --blob handling.

--- fiptool.c.orig	2019-05-14 04:51:36.000000000 +0000
+++ fiptool.c
@@ -271,10 +271,10 @@ static void uuid_from_str(uuid_t *u, con
 	    &u->node[2], &u->node[3],
 	    &u->node[4], &u->node[5]);
 	/*
-	 * Given the format specifier above, we expect 11 items to be scanned
+	 * Given the format specifier above, we expect 16 items to be scanned
 	 * for a properly formatted UUID.
 	 */
-	if (n != 11)
+	if (n != 16)
 		log_errx("Invalid UUID: %s", s);
 }
 
